features/sync_spec.rb

```diff
@@ -0,0 +1,13 @@
+require File.expand_path('../support/command_rspec_helper', __FILE__)
+
+describe 'rmt-cli' do
+  describe 'sync' do
+    command '/usr/bin/rmt-cli sync'
+    its(:exitstatus) { is_expected.to eq 0 }
+    its(:stdout) { is_expected.to include("INFO -- : Cleaning up the database\n") }
+    its(:stdout) { is_expected.to include("INFO -- : Downloading data from SCC\n") }
+    its(:stdout) { is_expected.to include("INFO -- : Updating products\n") }
+    its(:stdout) { is_expected.to include("INFO -- : Updating repositories\n") }
+    its(:stdout) { is_expected.to include("INFO -- : Updating subscriptions\n") }
```

Comment:
I don't think we should care about the output of the command in the integration test at all:
 * If we want to check the output of the command, it would be much easier to test this in unit tests (no need to have this super complicated setup with docker).
 * If the output is changed by a PR, the tests in the PR would be green and would break in the Jenkins pipeline after the PR is merged to master.

What we actually should verify is that SCC sync works correctly, which means that we can get products, repositories, etc. downloaded to the local database.

Reply:
fixed